packages/api/routes/website.js

```diff
@@ -0,0 +1,17 @@
+const { Router } = require("express");

```

Comment:
@soumyadip007 Didn't we decide to call these as `web properties` rather than `websites`?